### PR TITLE
FI-1736: Dev Improvements

### PR DESCRIPTION
**Pull request by Jammjammjamm**
# Summary
This branch updates the template to use the (not yet released) new methodology for running inferno.

# Testing Guidance
Make sure the updated readme instructions are correct.

Instructions for upgrading a test kit repo to use this functionality are located in [the draft release notes](https://github.com/inferno-framework/inferno-template/releases). You can test these out on some of our test kit repos.

### CODE REVIEW

**Comment by yunwwang** [comment]
Followed the instruction in READ.me and successfully initialized inferno_template instance.

**Comment by Jack-Fraser16** [comment]
I'm getting the following trying to run using ./setup.sh
```
ERROR: Cannot extend service 'fhir_validator_app' in /Users/jfraser/Desktop/Github/inferno-template/docker-compose.background.yml: services with 'depends_on' cannot be extended
ERROR: Cannot extend service 'fhir_validator_app' in /Users/jfraser/Desktop/Github/inferno-template/docker-compose.background.yml: services with 'depends_on' cannot be extended
ERROR: Cannot extend service 'fhir_validator_app' in /Users/jfraser/Desktop/Github/inferno-template/docker-compose.background.yml: services with 'depends_on' cannot be extended
```

**Comment by Jammjammjamm** [comment]
I don't see that error. What version of docker-compose do you have?
```
ᐅ docker-compose --version
Docker Compose version v2.10.2
```

**Comment by Jack-Fraser16** [comment]
> I don't see that error. What version of docker-compose do you have?
> 
> ```
> ᐅ docker-compose --version
> Docker Compose version v2.10.2
> ```

That was it

File: README.md [hunk 1]
```diff
@@ -3,27 +3,152 @@
...
+[3] pry(main)> suite.groups.map(&:title)
+=> ["Capability Statement", "Patient  Tests"]
+[4] pry(main)> suite.groups.first.tests.map(&:title)
+=> ["Read CapabilityStatement"]
+```
+
+It is also possible to set a breakpoint using the [debug
+gem](https://github.com/ruby/debug) within a test's `run` block to debug test
+behavior:
+- Add `require 'debug/open_nonstop'` and `debugger` to set the breakpoint.
```
Comment (Jack-Fraser16) [inline README.md:81]: This is awesome.

File: README.md [hunk 2]
```diff
@@ -3,27 +3,152 @@
...
+  verb=
+instance variables: @created_at  @direction  @headers  @id  @index  @name  @request_body  @response_body  @result_id  @status  @test_session_id  @updated_at  @url  @verb
+(ruby:remote) request.status
+200
+(ruby:remote) request.response_body
+"{\n  \"resourceType\": \"Patient\" ... }"
+(rdbg:remote) ?    # help command
+
+### Control flow
+
```
Comment (Jack-Fraser16) [inline README.md:147]: Should this be `[step]`?

**Review by Jack-Fraser16 — APPROVED** [review]
Works as expected - README updates are superb

**Pull request merged**